<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(100, 100, 100);</string>
  </property>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>331</width>
     <height>191</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">alternate-background-color: rgb(74, 74, 74);
font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(182, 182, 182);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="GotoMenuButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>20</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(80, 80, 80);
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Return to Menu</string>
   </property>
  </widget>
  <widget class="QPushButton" name="gototaskbutton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>210</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(80, 80, 80);
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Add Task</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="listWidgetTaskList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>341</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(125, 125, 125);
font: 9pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <column>
    <property name="text">
     <string>Tasks</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="RemoveTask">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>250</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(80, 80, 80);
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Remove Task</string>
   </property>
  </widget>
  <zorder>listWidgetTaskList</zorder>
  <zorder>calendarWidget</zorder>
  <zorder>GotoMenuButton</zorder>
  <zorder>gototaskbutton</zorder>
  <zorder>RemoveTask</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
